centreon/tests/e2e/features/Platform-upgrade-update/01-platform-update/index.ts

```diff
@@ -124,6 +124,12 @@ Given(
                 break;
               case 'penultimate stable':
                 minor_version_index = stable_minor_versions.length - 2;
+                if (
+                  stable_minor_versions[minor_version_index] ===
+                  Cypress.env('lastStableMinorVersion')
+                ) {
+                  return cy.stopContainer({ name: 'web' }).wrap('skipped');
+                }
```

Comment:
you should update conditon line 140 directly instead of duplicate code
(remove condition line 118 too)